### DISCUSSION TITLE
getFrequencyError() compile problem? (solved)

### DISCUSSION THREAD

**Discussion by stardad60** [jgromes/RadioLib#713] [Q&A]
Just discovered this useful library.  Started using the simple receive and transmit examples for a sx1262 based Heltec V3 board in the Arduino 2.04 IDE. The compile fails not finding a getFrequencyError() member .   I can comment this line out  and the compile then does fine and I can transmit and recieve LoRa packets. I tried a clean install and got the same error. I certainly can see the getFrequencyError() function in the 1262.cpp file so I'm not sure what is going on. Maybe it's an issue but since I'm new with this library, it's probably me. I'm not the best at tracing through libraries though  and need a little help. I  would include the compile output  but it exceeds the maximum characters. 
I did see on the wiki this note attached to this member: ....This functionality is based on [SX128x](https://jgromes.github.io/RadioLib/class_s_x128x.html) implementation and not documented on [SX126x](https://jgromes.github.io/RadioLib/class_s_x126x.html). While it seems to be working, it should be used with caution!

 Any hints?

**Comment by jgromes** [comment]
I checked compilation in Arduino IDE and there seems to be no problem. Are you sure you have the latest version of the library? Which one is it and where did you download it from? This functionality was added quite recently.

**Reply by stardad60** [reply]
Took it from the Arduino Library Manager. Version 5.7.

**Reply by stardad60** [reply]
As I mentioned, I'm not always great a tracing thru libraries, but at my 2nd attempt I did a search in the library under my Arduino directory  and  I see that the getFrequency Error method is in the 127x and 128x cpp and h files but not in the 126x.  That makes sense so now I don't understand  why that is.

**Reply by stardad60** [reply]
Installed 5.6 version of library, not there.  Reinstalled 5.7 version of library, still not there.

**Reply by jgromes** [reply]
Frequency error reading for SX126x was added in [this commit](https://github.com/jgromes/RadioLib/commit/cfe6128656fb9fe6cc2a98c8b36bb9a99e3175be) last week. 5.7.0 was released three weeks ago, hence you don't have the new functionality.

Did you take the example from GitHub, or from Arduino IDE examples?

**Reply by stardad60** [reply]
Ah Ha!   I did not originally see the Arduino IDE examples so I did indeed  take them from GitHub.   Now going back,  I  do see the examples in the IDE.   Comparing the files,  I see the differences, now I understand.  I knew it was likely me. Thank You very much. Solved.

**Reply by jgromes** [reply]
Glad I could help! A new version of the library with all recent additions and fixes should be published in a week or two.
